### PR TITLE
Add initial README and contribution/governance model

### PR DESCRIPTION
**Pull request by GregSutcliffe**
As per earlier discussion. In particular, I hope the governance model addresses @samccann's concerns :)

### CODE REVIEW

**Comment by cybette** [comment]
LGTM

**Review by cybette — APPROVED** [review]

**Comment by GregSutcliffe** [comment]
Inline what what the doc, says, I have updated the GitHub protections to require 2 reviews :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+This is still a work in prgress, and needs to be ratified
+
+## Principles
+
+This repo adheres to the following principles:
+
+- Open: Contribution is always welcome.
+- Respectful: See the [Code of Conduct](CODE_OF_CONDUCT.md).
+- Transparent and accessible: Work and collaboration should be done in public.
+  See Governance, below.
```
Comment (samccann) [inline CONTRIBUTING.md:10]:
```suggestion
  See [Governance](#governance) section for details.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+## How to contribute
+
+We are very happy to receive contributions from the community in any form, but in particular we would love:
+- Blog posts
+- New resources
+- Typos / fixes
+- Style improvements
+
+Please use a GitHub pull reuqest to submit your contributions. If you have a
```
Comment (samccann) [inline CONTRIBUTING.md:22]:
```suggestion
Please use a GitHub pull request to submit your contributions. If you have a
```

**Review by samccann — APPROVED** [review]
Fixed a few spelling nits

**Comment by GregSutcliffe** [comment]
thanks @samccann! @Andersson007 @anweshadas could you take a look?

**Review by cybette — APPROVED** [review]

**Comment by GregSutcliffe** [comment]
Fixed conflict from #2 and squashed the typos into the main commit.

**Review by oraNod — APPROVED** [review]

**Comment by GregSutcliffe** [comment]
thanks for the review folks. For the record we should properly ratify this once we get the full WG up and running.

**Pull request merged**